README.md

```diff
@@ -52,91 +52,83 @@ Implementing a table involves three component types- `<Table/>`,`<Column/>`, and
     headerHeight={50}
     ...
   </Table>
-```    
+```
 
 `<Column />` defines the way data is displayed for one column in the table, including all cell behavior for that column. Rather than manipulating each cell directly, pass a cell component as a prop to the column, and the column will render a cell for each index in the data array.
 
 ```javascript
-    <Column
-      header={<Cell>Col 1</Cell>}
-      cell={<Cell>Column 1</Cell>}
-      width={2000}
-    />
+<Column header={<Cell>Col 1</Cell>} cell={<Cell>Column 1</Cell>} width={2000} />
 ```
+
 The cell components in a column will receive the current array index of your data as a prop (`this.props.rowIndex`). Use this to access the correct value for each cell.
+
 ```javascript
-    const rows = [0,1,2];
-    
-    <Column
-      header={<Cell>Column 1</Cell>}
-      cell={({rowIndex, ...props}) => (
-        <Cell {...props}>
-        {rows[rowIndex]}
-        </Cell>
-      )}
-      width={2000}
-    />
+const rows = [0, 1, 2];
+
+<Column
+  header={<Cell>Column 1</Cell>}
+  cell={({ rowIndex, ...props }) => <Cell {...props}>{rows[rowIndex]}</Cell>}
```

Comment:
For now, let's leave mark down files out of prettier's hands as some of the changes here seem subjective. 

We can have a separate PR to focus on these later.